osc-server/src/main/java/org/osc/core/server/Server.java

```diff
@@ -349,11 +351,16 @@ private void startScheduler() throws SchedulerException {
 
         JobDetail syncDaJob = JobBuilder.newJob(SyncDistributedApplianceJob.class).usingJobData(jobDataMap).build();
         JobDetail syncSgJob = JobBuilder.newJob(SyncSecurityGroupJob.class).usingJobData(jobDataMap).build();
-        Trigger syncDaJobTrigger = TriggerBuilder.newTrigger().startNow().withSchedule(SimpleScheduleBuilder
-                .simpleSchedule().withIntervalInMinutes(this.scheduledSyncInterval).repeatForever()).build();
 
-        Trigger syncSgJobTrigger = TriggerBuilder.newTrigger().startNow().withSchedule(SimpleScheduleBuilder
-                .simpleSchedule().withIntervalInMinutes(this.scheduledSyncInterval).repeatForever()).build();
+        // TODO: Remove the delay, once plugin state listener is implemented.
+        // Related issue: https://github.com/opensecuritycontroller/osc-core/issues/545
+		Trigger syncDaJobTrigger = TriggerBuilder.newTrigger()
+				.startAt(new Date(System.currentTimeMillis() + SERVER_SYNC_DELAY)).withSchedule(SimpleScheduleBuilder
```

Comment:
You can do this to make it more readable...

Date.from(Instant.now().plusSeconds(SERVER_SYNC_DELAY)))